docs/installation/extending-odc.rst

```diff
@@ -254,6 +257,17 @@ The memory index driver may be useful:
 3. as an example/template for developing other index drivers.
 
 
+Postgis Implementation
```

Comment:
PostGIS

Reply:
Yes, "other bits and pieces" being mostly a slight refactor of the integration test fixtures to allow tests to be run against both drivers.